ivy.el

```diff
@@ -58,94 +58,128 @@
      :background "#1a4b77" :foreground "white")
     (((class color) (background dark))
      :background "#65a7e2" :foreground "black"))
-  "Face used by Ivy for highlighting the current match.")
+  "Face used by Ivy for highlighting the current match."
+  :group 'ivy-faces)
```

Comment:
There's no need for `:group 'ivy-faces` as the value of the `:group` tag defaults to the last `defgroup` in the current file. There's even a trend in the Emacs sources of removing redundant `:group` tags.